### DISCUSSION TITLE
Github critical advisory CVE-2019-17426 doesn't consider mongoose v4.13.21 ok, but it looks like it has been patched

### DISCUSSION THREAD

**Discussion by d1manson** [Automattic/mongoose#10889] [Q&A] [answered]
This advisory  https://github.com/advisories/GHSA-8687-vv9j-hgph says all versions less than 5.7.5 are vulnerable, but it seems that the patch went into v4.13.21. If that's correct, how can we get the advisory updated to not flag for that version?

See [v4.13.21](https://github.com/Automattic/mongoose/releases/tag/4.13.21) which mentions "Backport fix for [#8222](Automattic/mongoose#8222)". That is the issue linked to from the advisory, and it looks like the backported change is indeed the same as in v5.

Thanks!

**Comment by d1manson** [comment, marked as answer]
@vkarpov15 it looks like you put in the fix. perhaps you would know 
https://github.com/Automattic/mongoose/commit/f3eca5b94d822225c04e96cbeed9f095afb3c31c

**Reply by vkarpov15** [reply]
@d1manson I sent an email to nvd@nist.gov to see if they can update that page.

What is preventing you from upgrading to 5.x?

**Reply by d1manson** [reply]
thanks - we would like to upgrade to 5.x but we don't have quite the test coverage over all the places we use mongoose to be ready to do so.
